pkgs/faf-client/default.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  makeWrapper,
+  makeDesktopItem,
+  # regular jdk doesnt work due to problems with JavaFX even with .override { enableJavaFX = true; }
+  openjdk17-bootstrap,
+  xorg,
+  libGL,
+  gtk3,
+  glib,
+  alsa-lib,
+}: let
+  pname = "faf-client";
+  version = "2023.1.2";
+  src = builtins.fetchTarball {
+    url = "https://github.com/FAForever/downlords-faf-client/releases/download/v${version}/faf_unix_${builtins.replaceStrings ["."] ["_"] version}.tar.gz";
+    sha256 = "0lfyfbvjzfgkq3b485d8ma8fjc3hbc5p5h2ffl3lrvai6ksa59g2";
+  };
+  meta = with lib; {
+    description = "Official client for Forged Alliance Forever";
+    homepage = "https://github.com/FAForever/downlords-faf-client";
+    license = licenses.mit;
+  };
+  icon = builtins.fetchurl {
```

Comment:
Is this subject to change? We could name it `faf-client-${sha256}.png` to avoid any name clashes, should the hash change in the future.

Reply:
actually, the desktop file uses the store path for the icon, so it should be fine